src/server/debug.rs

```diff
@@ -339,6 +339,43 @@ impl Debugger {
                 None => Err(Error::NotFound("No store ident key".to_owned())),
             })
     }
+
+    pub fn modify_tikv_config(
+        &self,
+        module: &str,
+        config_name: &str,
+        config_value: &str,
+    ) -> Result<()> {
+        match module {
+            "rocksdb" | "raftdb" => {
```

Comment:
I think you can write `let db = if ... {} else if ... {} else { return Err }` to reduce indent.